libs/external-router/README.md

```diff
@@ -2,6 +2,22 @@
 
 ## Purpose
 
-This package extends `@angular/router` allowing you to wire an external router into the Router.
+This package extends `@angular/router` and allows you to add external systems that resolve routes, and then use the [Router](https://angular.io/guide/router) to navigate between them.
```

Comment:
This is a runon sentence

Reply:
https://github.com/graycoreio/daffodil/pull/1240/commits/e84f901398d30c4af371fb257714725339f3640a